packages/datadog-plugin-http/src/index.js

```diff
@@ -1,6 +1,242 @@
 'use strict'
 
-const client = require('./client')
-const server = require('./server')
+const url = require('url')
+const Plugin = require('../../dd-trace/src/plugins/plugin')
+const { storage } = require('../../datadog-core')
+const analyticsSampler = require('../../dd-trace/src/analytics_sampler')
+const tags = require('../../../ext/tags')
+const formats = require('../../../ext/formats')
+const urlFilter = require('../../dd-trace/src/plugins/util/urlfilter')
+const log = require('../../dd-trace/src/log')
+const web = require('../../dd-trace/src/plugins/util/web')
 
-module.exports = [].concat(client, server)
+const HTTP_HEADERS = formats.HTTP_HEADERS
+const SERVICE_NAME = tags.SERVICE_NAME
+const HTTP_STATUS_CODE = tags.HTTP_STATUS_CODE
+const HTTP_REQUEST_HEADERS = tags.HTTP_REQUEST_HEADERS
+const HTTP_RESPONSE_HEADERS = tags.HTTP_RESPONSE_HEADERS
+
+class HttpPlugin extends Plugin {
+  static get name () {
+    return 'http'
+  }
+
+  constructor (...args) {
+    super(...args)
+
+    this.addSub('apm:httpServer:request:start', ([req, res]) => {
+      if (this.config.server === false) return this.skip()
```

Comment:
Splitting the plugin in 2 separate plugins would allow to instead disable the client or server individually.

@bengl Is the `name` property important or can a plugin be split in 2?